shell/browser/electron_permission_manager.cc

```diff
@@ -233,7 +233,7 @@ void ElectronPermissionManager::RequestPermissionsWithDetails(
                 render_frame_host->GetProcess()->GetDeprecatedID());
       } else if (permission_type == blink::PermissionType::GEOLOCATION) {
         if (IsGeolocationDisabledViaCommandLine()) {
-          results.push_back(content::PermissionResult(
+          results.emplace_back(content::PermissionResult(
               blink::mojom::PermissionStatus::DENIED,
               content::PermissionStatusSource::UNSPECIFIED));
```

Comment:
```suggestion
          results.emplace_back(
              blink::mojom::PermissionStatus::DENIED,
              content::PermissionStatusSource::UNSPECIFIED);
```

IIRC, `emplace_back` should just take the constructor arguments directly, right?